pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-efs/guide.fr-fr.md

```diff
@@ -0,0 +1,409 @@
+
+Because your nodes are configured to be routed by the private network gateway, you need to add the gateway IP address to the ACLs.
+
+By using Public Cloud Gateway through our Managed Kubernetes Service, Public IPs on nodes are only for management purposes: [MKS Known Limits](/pages/public_cloud/containers_orchestration/managed_kubernetes/known-limits)
+
+You can get your OVHcloud Internet Gateway's Public IP by navigating through the OVHcloud Control Panel:
+
+`Public Cloud`{.action} > Select your tenant > `Network / Gateway`{.action} > `Public IP`{.action}
+
+You can also get your OVHcloud Internet Gateway's Public IP by using our APIs:
+
+> [!api]
+>
+> @api {v1} /cloud GET  /cloud/project/{serviceName}/region/{regionName}/gateway/{id}
+>
+
+You can find more details about how to use OVHcloud APIs with this guide: [First Steps with the OVHcloud APIs](/pages/manage_and_operate/api/first-steps)
+
+If you want to use your Kubernetes cluster to know your Gateway Public's IP, you can run these commands:
+
+```bash
+kubectl run get-gateway-ip --image=ubuntu:latest -i --tty --rm 
+```
+
+This command will create a temporary pod and open a console.
+
+You may have to wait a bit to let the pod be created. Once the shell appears, you can run this command:
+
+```bash
+apt update && apt upgrade -y && apt install -y curl && curl ifconfig.me
+```
+
+The Public IP of the Gateway you're using should appear.
+
+Click on the `Manage Access`{.action} menu of our newly created partition:
+
+![Manage Access of the EFS Volume](images/manage-efs-volume-access.png){.thumbnail}
+
+Add either the nodes' IPs one by one or the Gateway's Public IP depending on your configuration:
+
+![Allow nodes IP to access the EFS Volume](images/manage-efs-volume-access-ip.png){.thumbnail}
+
+You should now have something similar to this:
+
+![EFS Configuration](images/manage-efs-volume-acl-list.png){.thumbnail}
+
+### Step 2 - Configuring Kubernetes to use our newly created EFS volume
+
+Your Kubernetes cluster needs some additionnal piece of software to make use of the EFS volume. We will install those and then create a first volume, shared accross multiple pods.
+
+To do so, you can install the [csi-driver-nfs](https://github.com/kubernetes-csi/csi-driver-nfs):
+
+```bash
+helm repo add csi-driver-nfs https://raw.githubusercontent.com/kubernetes-csi/csi-driver-nfs/master/charts
+helm install csi-driver-nfs csi-driver-nfs/csi-driver-nfs --namespace kube-system --version v4.7.0 --set driver.name="nfs2.csi.k8s.io" --set controller.name="csi-nfs2-controller" --set rbac.name=nfs2
+```
+
+```console
+$ helm install csi-driver-nfs -n kube-system csi-driver-nfs/csi-driver-nfs --version v4.7.0 --set driver.name="nfs2.csi.k8s.io" --set rbac.name=nfs --set controller.name="csi-nfs2-controller"
+NAME: csi-driver-nfs
+LAST DEPLOYED: Mon Dec 16 16:13:31 2024
+NAMESPACE: kube-system
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+The CSI NFS Driver is getting deployed to your cluster.
```

Comment:
Is this part of the code snippet ? Or is it missing its closing ``` ?

Reply:
This is part of the code snippet. The closing "```" is written at the line 146